Packs/ShiftManagement/Playbooks/Shift_handover.yml

```diff
@@ -0,0 +1,1663 @@
+          "y": 1420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "32":
+    id: "32"
+    taskid: 3eaf1151-7598-4efd-84aa-772330c08cbb
+    type: condition
+    task:
+      id: 3eaf1151-7598-4efd-84aa-772330c08cbb
+      version: -1
+      name: Should get notification by Email?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "35"
+      "yes":
+      - "33"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEmpty
+          left:
+            value:
+              simple: 'inputs.ShiftManagerBriefing '
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": -160,
+          "y": 1570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "33":
+    id: "33"
+    taskid: f0eefd3d-efad-4d44-80e3-41da27a713cb
+    type: collection
+    task:
+      id: f0eefd3d-efad-4d44-80e3-41da27a713cb
+      version: -1
+      name: Get notification
+      type: collection
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "34"
+    scriptarguments:
+      execution-timeout:
+        simple: "0"
+    separatecontext: false
+    sla:
+      hours: 0
+      days: 0
+      weeks: 0
+    view: |-
+      {
+        "position": {
+          "x": -450,
+          "y": 1740
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+        simple: ${inputs.SOCManagerEmail}
+      subject:
+        simple: Briefing for the SOC team
+      body:
+        simple: Enter the URL and submit your briefing for the SOC team.
+      methods:
+      - email
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+      replyOptions:
+      - "Yes"
+      - "No"
+      defaultOption: "Yes"
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: What is your briefing for the team?
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: longText
+        options: []
+        optionsarg: []
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: SOC manager briefing
+      description: Enter the URL and provide the briefing for the team.
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+  "34":
+    id: "34"
+    taskid: 4c8871e0-9c90-4758-846b-d09c156c67f2
+    type: regular
+    task:
+      id: 4c8871e0-9c90-4758-846b-d09c156c67f2
+      version: -1
+      name: Set SOC manager briefing for the team
+      description: commands.local.cmd.set.incident
```

Comment:
@ShirleyDenkberg this is a server default description and not the correct description.